iaso/api/instances/instances.py

```diff
@@ -204,7 +204,10 @@ def attachments(self, request):
         instances = instances.for_filters(**filters)
         queryset = InstanceFile.objects_with_file_extensions.filter(instance__in=instances)
 
-        image_only = ImageOnlySerializer(data=request.query_params).get_image_only(raise_exception=False)
+        image_only_serializer = ImageOnlySerializer(data=request.query_params)
+        image_only_serializer.is_valid(raise_exception=True)
+        image_only = image_only_serializer.validated_data["image_only"]
```

Comment:
I fail to see how this is better than what was there before.
The code is a lot more verbose and you hardcoded `"image_only"`.

If you wanted to raise an exception, you could have done:
```
image_only = ImageOnlySerializer(data=request.query_params).get_image_only(raise_exception=True)
```